apex/contrib/csrc/multihead_attn/strided_batched_gemm.cuh

```diff
@@ -19,13 +19,13 @@
 
 // symbol to be automatically resolved by PyTorch libs
 /*
-rocblas_datatype a_type       = rocblas_datatype_f16_r; // OK
-rocblas_datatype b_type       = rocblas_datatype_f16_r; // OK
-rocblas_datatype c_type       = rocblas_datatype_f16_r; // OK
-rocblas_datatype d_type       = rocblas_datatype_f16_r;
-rocblas_datatype compute_type       = rocblas_datatype_f32_r;
+rocblas_datatype a_type       = HIPBLAS_R_16F; // OK
+rocblas_datatype b_type       = HIPBLAS_R_16F; // OK
+rocblas_datatype c_type       = HIPBLAS_R_16F; // OK
+rocblas_datatype d_type       = HIPBLAS_R_16F;
+rocblas_datatype compute_type       = HIPBLAS_R_32F;
 
-rocblas_gemm_algo algo           = rocblas_gemm_algo_standard;
+rocblas_gemm_algo algo           = HIPBLAS_GEMM_DEFAULT;
 int32_t           solution_index = 0;
 rocblas_int       flags          = 0;
 */
```

Comment:
Functions
convertTransToCublasOperation() and hipOperationToRocOperation() still used?

Reply:
This is under the comments. Not part of the code. Can remove it if needed.